### PR TITLE
Improve airflow configuration

### PR DESCRIPTION
**Pull request by feluelle**
# Description

## What is the current behavior?

Currently, our airflow configuration consists of settings in the config file as well as in env vars. This PR addresses it by using a single approach.

closes: https://github.com/astronomer/astro-sdk/issues/1219

## What is the new behavior?

We only use env vars for setting airflow configuration. Unfortunately every time we set those, we need to reload the modules for them to read the updated env var.

## Does this introduce a breaking change?

No.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: sql-cli/noxfile.py [hunk 1]
```diff
@@ -43,32 +43,20 @@ def test(session: nox.Session, airflow: str) -> None:
...
-
-    # At the moment flow run depends on the Airflow global Airflow Home - we need to fix this
-    # TODO: refactor so each test does this in their own sandboxed Airflow home
-    # From what I observed, the `run` command is the one using this, due to how the method
-    # utils.airflow.get_dag behaves under Airflow 2.2.0 (it defaults to $HOME/airflow)
-    airflow_home = f"~/sql-cli-python-{session.python}-airflow-{airflow}"
-    session.run(
-        "airflow", "db", "init", env={"AIRFLOW_HOME": airflow_home, "AIRFLOW__CORE__LOAD_EXAMPLES": "False"}
-    )
-
```
Comment (tatiana) [inline sql-cli/noxfile.py:62]: I'm so emotional about this code going away :sob:  Thank you so much, @feluelle !

File: sql-cli/sql_cli/utils/airflow.py [hunk 2]
```diff
@@ -24,61 +24,56 @@ def airflow_version() -> Version:
...
-    """
-    filename = airflow_home / "airflow.cfg"
-    parser = ConfigParser()
-    parser.read(filename)
-    parser["core"]["load_examples"] = "False"
-    parser["core"]["logging_level"] = "WARN"
-    with open(filename, "w") as out_fp:
-        parser.write(out_fp)
+    # Initialise the airflow database & hide all logs
+    os.system("airflow db init > /dev/null 2>&1")  # skipcq: BAN-B605,BAN-B607
```
Comment (tatiana) [inline sql-cli/sql_cli/utils/airflow.py:48]: Should we improve this in the verbose/debug ticket - so we can have more visibility if we use the flag?

**Review by tatiana — APPROVED** [review]
This looks incredible, @feluelle , amazing work! Thank you very much!!!

**Comment by feluelle** [comment]
Thank you ❤️

File: sql-cli/sql_cli/utils/airflow.py (hunk 2 above)
Comment (feluelle) [reply sql-cli/sql_cli/utils/airflow.py:48]: Yes, great idea! I will add this to the ticket.

**Pull request merged**